### PR TITLE
add 2wt regulated terminal

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: Abdelsalem <abdelsalem.hedhili@rte-france.com>

### CODE REVIEW

**Comment by AbdelHedhili** [comment]
There are a lot a warning triggered in the console when handling the creation form, any idea where does it come from ? I don't have a clue.

**Review by bendaamerahm — APPROVED** [review]
code review ok

File: src/translations/en.json [hunk 1]
```diff
@@ -432,6 +432,7 @@
 
     "CreateLine": "Create line",
     "Characteristics": "Characteristics",
+    "RegulatedSide": "regulated side",
```
Comment (archendel) [inline src/translations/en.json:435]:
```suggestion
    "RegulatedSide": "Regulated side",
```

File: src/translations/fr.json [hunk 2]
```diff
@@ -432,6 +432,7 @@
 
     "CreateLine": "Créer une ligne",
     "Characteristics": "Caractéristiques",
+    "RegulatedSide": "côté reglé",
```
Comment (archendel) [inline src/translations/fr.json:435]:
```suggestion
    "RegulatedSide": "Côté réglé",
```

**Review by archendel — CHANGES_REQUESTED** [review]
between "ratio tap changer" tab and "phase tap changer" tab, there is a small difference

when Voltage Regulation is unchecked, "regulated side" is marked (optional) in "phase tap changer" tab, but not in the other one

now testing full workflow

**Pull request closed without merging**